core/templates/pages/exploration-player-page/services/question-player-engine.service.ts

```diff
@@ -145,6 +145,15 @@ export class QuestionPlayerEngineService {
       errorCallback: () => void): void {
     this.contextService.setQuestionPlayerIsOpen();
     this.setAnswerIsBeingProcessed(false);
+    let currentIndex = questionDicts.length, randomIndex;
```

Comment:
For clarity, can we put `let randomIndex` on a new line?

Reply:
Done!